pylib/cqlshlib/copyutil.py

```diff
@@ -494,6 +497,7 @@ def make_params(self):
                     port=shell.port,
                     ssl=shell.ssl,
                     auth_provider=shell.auth_provider,
+                    parent_cluster=shell.conn if not IS_WINDOWS else None,
```

Comment:
This is kind of a concern... 
IMO fix in `DSE` is incomplete and will have unpredictable behavior on Windows,
but maybe supporting windows is not the case for Stargazer.